ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/private/cco-manual-workload-identity/provision/cucushift-installer-rehearse-azure-ipi-private-cco-manual-workload-identity-provision-chain.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/private/cco-manual-workload-identity/provision/cucushift-installer-rehearse-azure-ipi-private-cco-manual-workload-identity-provision-chain.yaml",
```

Comment:
If no special reason, suggest to use `azure/ipi/fullyprivate` to create the new install workflow.

Reply:
Ok, will use `azure/ipi/fullyprivate`.